code/modules/mob/living/simple_animal/friendly/ant.dm

```diff
@@ -0,0 +1,42 @@
+/mob/living/simple_animal/ant
```

Comment:
I'm surprised that these aren't hostile mob subtypes. Are player-controlled giant ants not intended to be able to attack things without a wumborian fugu gland boost?